vim/ftplugin/redtt.vim

```diff
@@ -79,12 +81,9 @@ function! s:EditWidth()
 endfunction
 
 function! s:CloseBuffer()
-  cclose
```

Comment:
what's this do?

Reply:
It closes the quickfix window, which is not currently opened by the redtt mode!

Reply:
ah! cool.